src/imagetilesource.js

```diff
@@ -114,8 +121,8 @@
             }
 
             $.addEvent(image, 'load', function () {
-                _this.width = image.naturalWidth;
-                _this.height = image.naturalHeight;
+                _this.width = $.getNatural(image).width;
+                _this.height = $.getNatural(image).height;
```

Comment:
@avandecreme you right, I make make fix more clear in last commit, test it in IE8 - it works.